crates/cxx-qt-build/src/lib.rs

```diff
@@ -402,7 +402,8 @@ impl CxxQtBuilder {
             .expect("Could not find Qt installation");
         qtbuild.cargo_link_libraries();
 
-        // Write cxx-qt-lib and cxx headers
+        // Write cxx-qt-gen, cxx-qt-lib and cxx headers
+        cxx_qt_gen::write_headers(format!("{header_root}/cxx-qt-gen"));
```

Comment:
think this actually collides with our normal output directory ? So it should probably be `cxx-qt` or `cxx-qt-common` or `cxx-qt-headers`

Reply:
Changed to `cxx-qt-common`